docs/graphql/manual/troubleshooting/index.rst

```diff
@@ -107,3 +107,17 @@ Solution
 - Delete the row from the ``hdb_permission`` table where the column ``table_name`` has the same value as the table
   mentioned in the error and the column ``perm_def`` involves the missing column.
 - Restart the GraphQL engine to verify.
+
+Error: cannot continue due to new inconsistent metadata
```

Comment:
Is this the exact error message? If not we should change it to that one.

Reply:
Yep. error msg is accurate